test/Closures/rlexe.xml

```diff
@@ -158,4 +158,10 @@
       <tags>BugFix</tags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>bug_OS_9008744.js</files>
+      <tags>BugFix,exclude_dynapogo</tags>
```

Comment:
Maybe we should do that?  I think we're not seeing a whole lot of deferred parse bugs these days anymore though.

In my ideal world our parser has no !buildAST unique code paths.  Only code paths for buildAST that do the actual AST construction.  All other logic is independent.  That would significantly reduce deferparse bugs.